src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -1584,6 +1584,13 @@ public FirstNameInfo GetInfo(FirstNameNode node)
             return _infoMap[node.Id] as FirstNameInfo;
         }
 
+        public DName? GetDisplayName(FirstNameNode node)
```

Comment:
What is this used for?

Reply:
Get the DisplayName of that node
If  *display* name is not the right term, let me know what to use. Logical? Virtual? Illlogical? ...